libs/ui-lib/lib/common/components/ui/formik/CodeField.tsx

```diff
@@ -83,6 +83,12 @@ const CodeField = ({
             language={language}
             onEditorDidMount={(editor) => setMonacoEditor(editor)}
             downloadFileName={downloadFileName}
+            onCodeChange={(value) => {
```

Comment:
Is this related to @batzionb 's comment in the `useEffect`?

Based on it, I think we should be able to remove the effect, as 
https://github.com/patternfly/patternfly-react/issues/7341 was merged some time ago.

Reply:
That is corret
You can remove the useEffect